scripts/release_scripts/deploy.py

```diff
@@ -273,6 +273,8 @@ def deploy_application_and_write_log_entry(
         version_to_deploy_to: str. The version to deploy to.
         current_git_revision: str. The current git revision.
     """
+    # Deploy cron yaml to GAE.
```

Comment:
Should probably move this to after the main app deployment

Reply:
Makes sense, done.